### ISSUE TITLE
crystal aura

### ISSUE THREAD

**Issue by basedguylandslug** [cornos/Cornos#91]
A crystal aura on the client would be pretty good, if you are already working on thius then sorry i guess

**Comment by feliscatusmeows** [comment]
We don't believe in alternative medicines like essential oils and crystals here.

**Comment by basedguylandslug** [comment]
the j

**Comment by Saturn5Vfive** [comment]
the j

**Comment by 0x150** [comment]
some time in the future ngl but not rn

**Comment by Doogie13** [comment]
This would take the client a long way if it were added

**Comment by Saturn5Vfive** [comment]
the j

**Comment by feliscatusmeows** [comment]
@Doogie13 no, we don't believe in crystal healing and their "auras", take your anti-vax things elsewhere.

**Comment by ajl223** [comment]
pee and cum too

**Comment by basedguylandslug** [comment]
what, you gonna cry? gonna piss your pants, maybe? maybe shit and cum?

**Comment by ajl223** [comment]
wait, wait, wait.... hold your horses... uhm... YOU'RE A GIRL GAMER?!!?! O_O Not to be a freak, but.. just when I thought you couldn't get more attractive.. you started playing video games. Nicely done, m'lady. You've just become every man's dream woman. If you had missed a couple before, now you can be sure you've got us ALL "drooling", lol.

**Comment by ajl223** [comment]
![image](https://user-images.githubusercontent.com/70416702/122660935-7926aa80-d153-11eb-964b-6d9ed401f4c5.png)
I wonder if this is against github tos
![image](https://user-images.githubusercontent.com/70416702/122660940-817ee580-d153-11eb-8b83-70a2c5b60759.png)

**Comment by 0x150** [comment]
what the fuck
